internal/distro/distro.go

```diff
@@ -67,7 +68,7 @@ var (
 func DiskByIDDir() string       { return diskByIDDir }
 func DiskByLabelDir() string    { return diskByLabelDir }
 func DiskByPartUUIDDir() string { return diskByPartUUIDDir }
-
+func ModprobeCmd() string       { return modprobeCmd }
```

Comment:
This looks good, but should be done for `vmur`, `cio_ignore` etc. For those they should probably be grouped together with a comment that they're for zVM.